enterprise/server/test/webdriver/invocation/invocation_test.go

```diff
@@ -2,18 +2,90 @@ package invocation_test
 
 import (
 	"context"
+	"flag"
 	"testing"
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/testutil/buildbuddy_enterprise"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/app"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/testbazel"
 	"github.com/buildbuddy-io/buildbuddy/server/testutil/webtester"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestAuthenticatedInvocation_LogUploadEnabled(t *testing.T) {
+var (
+	targetStr   = flag.String("webdriver_target", "local", "Target that should be tested by the webdriver.")
+	devSSOSlug  = flag.String("webdriver_dev_slug", "", "Dev SSO slug to login.")
```

Comment:
Instead of `local`, `dev`, `prod`, wdyt about having `local` and `remote`, where remote has to be explicitly configured with `--remote_app_endpoint` and `--remote_sso_slug`? I think it might be cleaner since this binary would only have to know about one k8s env at a time, instead of having the potential to accidentally communicate with multiple envs

Reply:
Done